src/tv2-common/migrations/sourceManifest.ts

```diff
@@ -26,9 +26,9 @@ export function MakeConfigForSources(
 				rank: 0
 			},
 			{
-				id: 'AtemSource',
-				name: 'ATEM input',
-				description: `ATEM vision mixer input for ${displayName} input`,
+				id: 'SwitcherSource',
+				name: 'Switcher input',
```

Comment:
Is the name `Switcher input` descriptive enough without the description under it? I'm asking because i honestly don't know if Switcher can relate to anything else than a Video Switcher. But it is of course in the description below.

Reply:
Changed to _Video Switcher_ everywhere